cypress/tests/ab-testing.cy.ts

```diff
@@ -0,0 +1,38 @@
+describe('A/B Testing Page', () => {
+  const PAGE_URL = 'abtest';
+
+  describe('Navigation', () => {
+    it('Should be linked correctly from home page', () => {
+      // Visit home page and click the correct link
+      cy.visit('/');
+      cy.get('a').contains('A/B Testing').click();
+
+      // Assert correct page loads
+      cy.url().should('contain', PAGE_URL);
+    });
+  });
+
+  describe('Functionality', () => {
+    beforeEach(() => {
```

Comment:
The beforeEach() here could be removed since this is a single-test describe() block. Instead of taking up 3 lines, it could just be used as the first step in the it()

Reply:
Done